internal/command/meta_backend_migrate.go

```diff
@@ -553,7 +553,7 @@ const inputBackendMigrateMultiToSingle = `
 The existing %[1]q backend supports workspaces and you currently are
 using more than one. The newly configured %[2]q backend doesn't support
 workspaces. If you continue, Terraform will copy your current workspace %[3]q
-to the default workspace in the target backend. Your existing workspaces in the
+to the default workspace in the destination backend. Your existing workspaces in the
```

Comment:
Perhaps we should be using "new backend" here, to line up with the other error messages above?